From 2777e0784354f687c862c8e25787b096bad2755b Mon Sep 17 00:00:00 2001
From: JasonHuang <Jason.Huang-EE@infineon.com>
Date: Thu, 9 Jun 2022 21:46:45 -0500
Subject: [PATCH 125/179] brcmfmac: Modified Kconfig help format

According to the kernel 5.10.9 kconfig-language.rst, the
attribute of help not support "---help---" any more.

Signed-off-by: JasonHuang <Jason.Huang-EE@infineon.com>
---
 drivers/net/wireless/broadcom/brcm80211/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/Kconfig b/drivers/net/wireless/broadcom/brcm80211/Kconfig
index 5dbc1e2a83fc..b520d46ffe5d 100644
--- a/drivers/net/wireless/broadcom/brcm80211/Kconfig
+++ b/drivers/net/wireless/broadcom/brcm80211/Kconfig
@@ -43,7 +43,7 @@ config BRCMFMAC_PCIE_BARWIN_SZ
 	depends on BRCMFMAC
 	depends on PCI
 	default n
-	---help---
+	help
 	  If you say Y here, the FMAC driver will use custom PCIE BAR
 	  window size. Say Y to allow developers to use custom PCIE
 	  BAR window size when HOST PCIE IP can support less then 4MB
-- 
2.17.1

